engine-modexp/src/arith.rs

```diff
@@ -399,19 +399,8 @@ pub fn join_as_double(hi: Word, lo: Word) -> DoubleWord {
 
 #[test]
 fn test_monsq() {
-    check_monsq(1, 31);
-    check_monsq(6, 31);
     // This example is intentionally chosen because 5 * 5 = 25 = 0 mod 25,
```

Comment:
I think it makes sense to keep this comment with the `check_monsq(5, 25)` test case.